src/main.py

```diff
@@ -0,0 +1,42 @@
+from leader import Leader
+from follower import Follower
+from tagrec import TagRecognition
+from inputcontroller import InputController
+
+
+"""
+Main script to run follower and leader functionality
+
+Takes input from camera to determine if current vehicle 
+should lead or follow based on detecting an AR tag.
+
+If an AR tag exists the follower functionality will run.
+If an AR tag does not exist the leader functionality will run.
```

Comment:
Change to

"If an ARTag does not exist, the vehicle will become the leader vehicle."